astro/src/content/docs/lifecycle/manage-users/user-actions.mdx

```diff
@@ -31,21 +31,23 @@ Below are the terms you'll encounter when working with Actions.
 * **Action** — An Action is a state or event that can be applied to a User. It is reusable for many Users in many Applications. Applying an Action to a specific User is called an Action instance. This is similar to programming, where you have classes (Actions) and objects (Action instances). An Action instance consists of one User applying the Action on another User, the time of the Action, and the name of the Action.
 
 <Aside type="note">
-Other FusionAuth documentation pages might speak interchangeably about Actions and Action instances. For example, saying "the Action expires" when it is actually the Action instance that expires. The pages call an Action instance "taking an Action" or "applying an Action", which is correct, but very wordy. In this guide, the detailed discussion of the two concepts requires distinguishing between them by using the word "instance". Actions and their instances are separate types of objects in FusionAuth, created in separate areas of the admin UI, and in separate APIs.
+Other FusionAuth documentation pages might speak interchangeably about Actions and Action instances. For example, saying "the Action expires" when it is actually the Action instance that expires. The pages call an Action instance "taking an Action" or "applying an Action", which is correct, but very wordy.
```

Comment:
This `Aside` belongs to the `Action`  item I think.. if so, can we indent this element to match? 

The way this is written right now, the `Aside` is left aligned with the `ul` so to speak instead of `* Action`. 

I think it should render like this:

![image](https://github.com/user-attachments/assets/0271ab16-e591-430f-8ddb-e1e5bf6cdd61)